app-emulation/docker/docker-1.11.2-r1.ebuild

```diff
@@ -0,0 +1 @@
+docker-9999.ebuild
```

Comment:
@coreos/team-os I don't know why I'm adding this file, it must've slipped in when rebasing/merging. Should I remove this file?